.eslintrc.cjs

```diff
@@ -15,7 +15,7 @@ module.exports = {
     ],
     "sonarjs/elseif-without-else": "warn",
     "sonarjs/no-duplicate-string": "warn",
-    "sonarjs/cognitive-complexity": "warn",
+    "sonarjs/cognitive-complexity": ["warn", 35],
```

Comment:
`35` allows most functions in the codebase, I could just add disabling comment to each one instead if preferred (there were around 10-ish or something I think)

I did not attempt on splitting the functions as I am not really familiar with the codebase here (but I am happy to give it a go if you want 🙂)

Reply:
I agree.
I need to take a closer look at the PR later today, I remember that there was a reason that it was this order. Might be old code